app/Http/Controllers/Auth/EmailVerificationPromptController.php

```diff
@@ -15,7 +15,7 @@ class EmailVerificationPromptController extends Controller
     public function __invoke(Request $request): RedirectResponse|View
     {
         return $request->user()->hasVerifiedEmail()
-                    ? redirect()->intended(route('dashboard', absolute: false))
+                    ? redirect()->intended(route('employee', absolute: false))
```

Comment:
please revert this to dashboard. we have implemented a dashboard for common purposes not only for employee

Reply:
Reverted all changes done to Dashboard... create a new issue to properly implement the dashboard